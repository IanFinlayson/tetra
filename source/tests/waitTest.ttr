def main():
    background t2:
        sleep(0.2)
        print("Done t2\n")
    background t1:
        sleep(0.1)
        print("Done t1\n")
    print("Launched them...\n")
    wait t1
    wait t2
    print("All done!")


