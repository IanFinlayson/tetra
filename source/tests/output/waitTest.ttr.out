Launched them...
Done t1
Done t2
All done!
